z3.ads

```diff
@@ -402,6 +404,18 @@ package Z3 is  --  GCOV_EXCL_LINE
 
    procedure Reset (Optimize : in out Z3.Optimize);
 
+   procedure Push (Optimize : in out Z3.Optimize);
+
+   procedure Pop (Optimize : in out Z3.Optimize);
+
+   function Get_Number_Of_Values (Optimize : Z3.Optimize) return Natural;
+
+   procedure Get_Values (Optimize  :     Z3.Optimize;
+                         Constants : out Int_Array;
+                         Values    : out Int_Array) with
+      Pre => Constants'Length = Get_Number_Of_Values (Optimize)
+             and then Values'Length = Get_Number_Of_Values (Optimize);
```

Comment:
Shouldn't also `Value'Length = Constants'Length`?

Reply:
This is implied since `Get_Number_Of_Values` is not volatile.